test/index.test.ts

```diff
@@ -4,10 +4,10 @@ import { Contract } from "ethers";
 import { FormatTypes, Interface, parseEther } from "ethers/lib/utils";
 import { ethers } from "hardhat";
 
-import { ToucanClient } from "..";
-import { IToucanCarbonOffsets } from "../typechain";
-import { poolTokenABI, swapperABI, tco2ABI } from "../utils/ABIs";
-import addresses from "../utils/addresses";
+import { ToucanClient } from "../dist";
```

Comment:
Is it expected that imports on code that uses the SDK as a dependency are expected to use `dist/`? I feel there should be a way to avoid that? cc @pheuberger

Reply:
Do you mean that the Lyra tests do it the same way? Maybe we cannot really import the SDK in our tests otherwise, which makes sense because it has to be published first.